resdk/resources/collection.py

```diff
@@ -44,37 +41,31 @@ def __init__(self, slug=None, id=None,  # pylint: disable=redefined-builtin
         self.descriptor = None
         #: descriptor schema
         self.descriptor_schema = None
-        #: collections
-        self.collections = None
 
-        BaseResource.__init__(self, slug, id, model_data, resolwe)
+        super(BaseCollection, self).__init__(slug, id, model_data, resolwe)
+
+    def update(self):
+        """Clear cache before updating."""
```

Comment:
First write about what this function does in general (copy from BaseResource). Write specifics below.